sky/packages/sky/lib/src/rendering/proxy_box.dart

```diff
@@ -172,24 +200,25 @@ class RenderAspectRatio extends RenderProxyBox {
   String debugDescribeSettings(String prefix) => '${super.debugDescribeSettings(prefix)}${prefix}aspectRatio: ${aspectRatio}\n';
 }
 
-class RenderShrinkWrapWidth extends RenderProxyBox {
-
-  // This class will attempt to size its child to the child's maximum
-  // intrinsic width, snapped to a multiple of the stepWidth, if one
-  // is provided, and given the provided constraints; and will then
-  // adopt the child's resulting dimensions.
-
-  // Note: laying out this class is relatively expensive. Avoid using
-  // it where possible.
-
-  RenderShrinkWrapWidth({
+/// Sizes its child to the child's intrinsic width
+///
+/// This class will size its child's width to the child's maximum intrinsic
+/// width. If [stepWidth] is non-null, the child's width will be snapped to a
+/// multiple of the [stepWidth]. Similarly, if [stepHeight] is non-null, the
+/// child's height will be snapped to a multiple of the [stepHeight].
+///
+/// Note: This class is relatively expensive. Avoid using it where possible.
```

Comment:
Give an example of when you _would_ use it (e.g., when the incoming constraints are infinitely wide, and the child would normally expand to fit the maxWidth, but you want the child to have a finite dimension).